tests/js/test-utils.js

```diff
@@ -30,24 +31,31 @@ import { createTestRegistry } from './utils';
  * @param {Object}   options               Render options.
  * @param {Function} options.setupRegistry A function which accepts the registry instance to configure it.
  * @param {Function} options.registry      A specific registry instance to use. Defaults to a fresh test registry with all stores.
+ * @param {boolean}  options.useRouter     Decides if wrapped with HashRouter component. Defaults to false, no router.
```

Comment:
I'm not a huge fan of this approach, as _eventually_ I'd think we'll want to use React Router for more and more of the app, eventually even allowing for switching between module pages without a fullpage reload. I'd prefer if having a Router enabled was assumed/default behaviour for tests, just like having a datastore is.

What's the drawback to always enabling a router?

I suppose I can see where you might want to conditionally enable a hash router as part of the render test, but in that case maybe this should be `hashRouter` or `useHashRouter` or even `wrapInHashRouter`.